jekyll/_cci2/local-cli.md

```diff
@@ -51,7 +53,7 @@ $ circleci config validate -c .circleci/config.yml
 ```
 The config validate command checks your local config.yml file for syntax errors.
 
-**Note**: Local jobs don’t cache dependencies. You may want to comment out dependency sections if you are testing YAML syntax. Or, as in this example, use the `-c` flag to specify a local `config.yml` file that doesn’t pull in large dependencies.
+***note:*** Local jobs don’t cache dependencies. You may want to comment out dependency sections if you are testing YAML syntax. Or, as in this example, use the `-c` flag to specify a local `config.yml` file that doesn’t pull in large dependencies.
```

Comment:
Same comment as above. I'd keep this capitalized and remove the extra asterisk.

A larger comment about "Notes": long notes are a bit of a smell to me. If there is so much to be called out explicitly, then arguably we should be putting that into the body.

Reply:
Didn't write this paragraph.

Reply:
Whoops! Yeah, it's tough to tell.